pages/index.js

```diff
@@ -1,7 +1,10 @@
+import TripList from "@/components/TripList";
+
 export default function HomePage() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>TravelApp</h1>
+      <TripList />
     </div>
```

Comment:
I would like to suggest you to change the generic `<div>` into a semantic `<main>` element.